sljit_src/sljitNativeARM_32.c

```diff
@@ -41,6 +41,69 @@ SLJIT_API_FUNC_ATTRIBUTE const char* sljit_get_platform_name(void)
 #endif
 }
 
+#define CPU_FEATURE_NEON	(1UL << 12)
+static unsigned long cpu_feature_list;
+
+#if (defined SLJIT_DETECT_SIMD && SLJIT_DETECT_SIMD)
+#if defined(HAVE_GETAUXVAL) || defined(HAVE_ELF_AUX_INFO)
+#include <sys/auxv.h>
+#endif
+
+#ifdef __NetBSD__
```

Comment:
might also apply to FreeBSD, specially when !HAVE_ELF_AUX_INFO but hadn't been able to test that yet